### PR TITLE
Update scenarios and add demo video

### PR DESCRIPTION
**Pull request by rahinisrinivas**

### CODE REVIEW

File: docs/rule-promotions/overview.md [hunk 1]
```diff
@@ -708,6 +708,421 @@ The following messages indicating the removal of Promotion B and the application
...
+                        "amount": -800,
+                        "currency": "USD",
+                        "includes_tax": false
+                    },
+                    "code": "10-off",
+                    "id": "c6ed29ea-be53-4e94-ba43-aa12d34b2124",
+                    "promotion_source": "rule-promotion",
+                    "is_cart_discount": true,
+                    "ordinal": 2
+                }
```
Comment (dazadep) [inline docs/rule-promotions/overview.md:765]: maybe just show the discounts in the doc. not sure how useful rest of the json is

Comment (rahinisrinivas) [reply docs/rule-promotions/overview.md:765]: makes sense. I was thinking what to show since the json is too big.

Comment (rahinisrinivas) [reply docs/rule-promotions/overview.md]: amended

**Review by davidmoltin — APPROVED** [review]

File: docs/rule-promotions/overview.md [hunk 2]
```diff
@@ -708,6 +708,319 @@ The following messages indicating the removal of Promotion B and the application
...
+                                "6683e778-54a1-4206-a042-36412ee0f104": {
+                                    "amount": -2000,
+                                    "currency": "USD",
+                                    "formatted": "-$20.00"
+                                }
+                            }
+                        }
+                    }
+                },
+                "timestamps": {
```
Comment (dazadep) [inline docs/rule-promotions/overview.md:755]: Oh I think we can remove everything below in the json response as well, as I don't think it is super interesting for this scenario

**Pull request merged**